src/components/Map.jsx

```diff
@@ -19,7 +19,7 @@ class Map extends Component {
     updateMap(this.map, nextProps.vehicles, terminals);
 
     if(this.props.orderStage === 'draft' && nextProps.orderStage === 'searching') {
-      initiateZoomTransition(this.map, nextProps.pickup, nextProps.pickup);
+      initiateZoomTransition(this.map, nextProps.pickup, nextProps.dropoff);
```

Comment:
Why this change?
It makes sense to focus on the pickup points because we would like to show the drones appear on the map as they bid... and the bidding drones should be the ones closest to the pickup location

Reply:
When I initially did it, used both pickup and dropoff, so I thought this line was a mistake I made.

Reply:
No, this was deliberate: https://github.com/DAVFoundation/missions/issues/55